_layouts/restdocs.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+---
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Docs - Data Commons</title>
+    <!-- needed for adaptive design -->
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@600&family=Roboto&display=swap" rel="stylesheet">
+
+    <!--
+    ReDoc doesn't change outer page styles
+    -->
+    <style>
+      body {
+        margin: 0;
```

Comment:
I'm not sure if this is necessary (margin:0 is already added by bootstrap and removing padding:0 in the console doesn't seem to have a visible difference).